### PR TITLE
Fix General UI issues

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#18917

*Description of changes:*
- Fix General UI issues

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/RegistrationsWrapper.vue [hunk 1]
```diff
@@ -1173,5 +1174,30 @@ export default defineComponent({
...
+      margin-right: 15px;
+    }
+    .v-icon--size-default {
+      // min-width: 14px;
+      // max-width: 14px;
+    }
+
+    .v-selection-control--density-compact {
+      // --v-selection-control-size: unset;
+    }
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/RegistrationsWrapper.vue]: Could probably remove these classes

File: ppr-ui/src/components/common/RegistrationsWrapper.vue [hunk 2]
```diff
@@ -1173,5 +1174,30 @@ export default defineComponent({
...
+  }
+
+  :deep(.v-list-item--density-default.v-list-item--one-line) {
+    height: 30px !important;
+    min-height: 30px !important;
+  }
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/RegistrationsWrapper.vue]: Best practices and such. 
Explore the overrides if possible, especially if this is standardized height size.

File: ppr-ui/src/components/common/RegistrationsWrapper.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/common/RegistrationsWrapper.vue]: Ooops yes, done

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 3]
```diff
@@ -1119,7 +1119,9 @@ export default defineComponent({
   opacity: 0.4 !important;
 }
 
-:deep(.home-owners-icon) {
+.home-owners-icon {
   vertical-align: middle;
+  width: 25px;
+  height: 21px;
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Saw this in another spot as well, if this is consistent with certain implementations of icons etc, a common class with a generic name would be ideal.

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by cameron-eyds — APPROVED** [review]
Looks great! Just some minor comments

**Review by RuoxuanPengBC — APPROVED** [review]
looks great!

File: ppr-ui/src/components/common/RegistrationsWrapper.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/components/common/RegistrationsWrapper.vue]: This needs to be cleaned up, i was trying to figure out which class was blocking the normal way to set the style. Definitely didn't mean to leave it as is. Thanks for catching!

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 3 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Updated

File: ppr-ui/src/components/common/RegistrationsWrapper.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/components/common/RegistrationsWrapper.vue]: Updated

**Comment by cameron-eyds** [comment]
@dimak1 Just realized you had this pointing at the feature branch and it wasn't in sync with main. 
This will need to be re-opened and pointed/merged to main, the feature branch was already folded in.

**Pull request merged**